circle.yml

```diff
@@ -31,7 +31,7 @@ test:
     - ./bin/run-mochitests-docker
   post:
     - yarn flow-coverage -- -t json -o $CIRCLE_TEST_REPORTS/flow-coverage
-    - yarn check
+    - yarn check --integrity
```

Comment:
calling this out as it's a friendlier check.

we might want to go back to yarn check after we jump to react 15.6 or 16. At the moment we're so far back it's hurting us